packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -73,7 +73,9 @@ class CupertinoSwitch extends StatefulWidget {
   /// change state until the parent widget rebuilds the switch with the new
   /// value.
   ///
-  /// If null, the switch will be displayed as disabled.
+  /// If null, the switch will be displayed as disabled with an opacity of
+  /// 0.5. This is opacity value appears to be accurate in comparison to the
```

Comment:
Can you put the 0.5 as a constant in this file? You can then refer to it in this doc as `If null, the switch will be displayed as disabled with an opacity of [kCupertinoSwitchDisabledOpacity]`. 

We don't need to put the second sentence in the field doc. It can just be a code comment on the constant itself.